src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/newinitiate/PostInitiateByIdControllerTest.java

```diff
@@ -324,6 +328,62 @@ void given_initiate_request_when_there_is_error_then_do_rollback() throws Except
         assertFalse(taskResourceRepository.getByTaskId(taskId).isPresent());
     }
 
+    @Test
+    void given_initiate_request_when_there_is_ccd_400_then_do_rollback() throws Exception {
+
+        Request request = Request.create(Request.HttpMethod.POST, "url",
+                                         new HashMap<>(), null, new RequestTemplate());
+
+        when(clientAccessControlService.hasExclusiveAccess(SERVICE_AUTHORIZATION_TOKEN))
+            .thenReturn(true);
+
+        when(caseDetails.getCaseType()).thenReturn("Asylum");
+        when(caseDetails.getJurisdiction()).thenReturn("IA");
+        when(caseDetails.getSecurityClassification()).thenReturn(("PUBLIC"));
+
+        FeignException exception = new FeignException.BadRequest(
+            "Downstream Dependency Error",
```

Comment:
Maybe give a better or more realistic error message here?  E.g. CCD caseId is in the incorrect format?

Reply:
I will make the changes.